### ISSUE TITLE
Porting to JP

### ISSUE THREAD

**Issue by vybre** [dice4321/Gunkolle#34]
If I wanted to use this with the JP version of the game would it be as simple as me replacing all the images for the JP ones or would additional steps be required as well?

**Comment by dice4321** [comment]
Additional steps would be required, while the majority of the program does use images to search and click, there are a few places where I use pixels locations to check and click. I've never played JP, so chances are, you would need to change all the [pixel colors](https://github.com/dice4321/Gunkolle/blob/master/Constants/PixelColor.ahk) and possibly the [location](https://github.com/dice4321/Gunkolle/blob/master/Functions/PixelMap.ahk) of where I check these pixels as well.

**Comment by dice4321** [comment]
For most images, general location is fine, but if you look into the code and you a line like this `WFindClick("DollList\"Doll%DollCount1%, "rNoxPlayer mc a125,125,-775,-220")` that means the image search location has been restricted in reference to the nox player, `125,125,-775,-220`, in `x,y,w,h` dimensions.

**Comment by vybre** [comment]
Alright, I'll spend the weekend and see if I can make heads or tails of it, thank you for replying!

**Comment by vybre** [comment]
Just an update, replacing some of the images from EN to JP was all that was needed for the most part. The doll images I didn't have to change at all.

I didn't have to change any of the pixel colors or locations, even though EN is using the older client it looks like they're similar enough.

I've only tested it on 4-3e so far, I haven't done 0-2 yet since it's harder to drag without armor being released onto the server.

One of things I did have to change was during combat when the script would look for the combatpause, the battle would finish and transition back to the map but before the script could pick up the endturn image a click would be pressed which would stop the planned route from happening. (It would show 0, a click would be pressed and then it would show 1 right after) I think this might have to do with the higher latency I have to the JP server and the way the client reacts to that but I can't be too sure, my solution was to change this line

`FindClick(A_ScriptDir "\pics\CombatPause", "rNoxPlayer mc o5 Count1 n0 w30000,50")`

to

`FindClick(A_ScriptDir "\pics\CombatPause", "rNoxPlayer mc o5 Count1 n0 w30000,200")`

about 100+ runs so far and no issues.
